posts/2022/sparse-PFT-gridding.ipynb

```diff
@@ -0,0 +1,12504 @@
+{
```

Comment:
<p>Maybe a brief overview about the model data... not too in-depth, but a brief intro of what CLM is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Yeah agree. Lets see if I can find someone to contribute this stuff (oceanographer on unfamiliar land here)</p>